modules/core/test/src/alloy/validation/SimpleRestJsonTimestampFormatValidatorSpec.scala

```diff
@@ -74,12 +74,24 @@ final class SimpleRestJsonTimestampFormatValidatorSpec extends munit.FunSuite {
         .builder()
         .id("SimpleRestJsonTimestamp")
         .severity(Severity.WARNING)
+        .shape(member1)
         .message(
-          "test#time: Timestamp shape test#time does not have a timestamp format trait "
+          "A Timestamp shape  does not have a timestamp format trait"
         )
-        .build()
+        .build(),
+      ValidationEvent
+        .builder()
+        .id("SimpleRestJsonTimestamp")
+        .shape(member0)
+        .severity(Severity.WARNING)
+        .message(
+          "A Timestamp shape  does not have a timestamp format trait"
+        )
+        .build(),
+
     )
     assertEquals(result.size, 2)
```

Comment:
I think you should drop the size check, the only thing it does is hinder debugging when a failure occurs. `0 != 1` is much less helpful than `List(something) != List.empty`

wdyt?

Reply:
same for other checks below